package.json

```diff
@@ -22,13 +22,13 @@
   },
   "homepage": "https://github.com/massiveart/stylelint-config-ma#readme",
   "dependencies": {
-    "stylelint-config-standard": "^18.2.0",
+    "stylelint-config-standard": "^20.0.0",
     "stylelint-scss": "^3.0.0"
   },
   "devDependencies": {
-    "stylelint": "^10.0.0"
+    "stylelint": "^13.0.0"
   },
   "peerDependencies": {
-    "stylelint": "^8.3.0 || ^9.0.0 || ^10.0.0"
+    "stylelint": "^10.1.0 || ^11.0.0 || ^12.0.0 || ^13.0.0"
```

Comment:
why not allow the lower versions we did normally just add the new stylelint versions?

Reply:
@alexander-schranz do we realy need to support node version <10 with a new version? `stylelint-config-standard` dropt support for node version <10 with release `19.0.0` and this PR was made to get rid of the warnings that  `stylelint-config-standard` version `18.2.0` does no support node version >10.0.0

Reply:
> do we realy need to support stylelint version

else we would need todo a new major release which will actually every project then update to it. So the question is more does it hurt to support older stylelint versions?

Reply:
@alexander-schranz I will check.

Reply:
@alexander-schranz I added the older versions back. Not 100% sure how `peerDependencies` but I think it should work.